skema/skema-rs/mathml/src/ast/operator.rs

```diff
@@ -100,30 +121,28 @@ pub enum Operator {
     Arcsec,
     Arccsc,
     Arccot,
+    /// Mean operator
     Mean,
     /// Summation operator
-    Sum,
-    /// Summation operator with lowlimit and uplimit
-    SumUnderOver(SumUnderOver),
+    Summation(Summation),
+    /// Cross product operator
     Cross,
     /// Hat operator, e.g. \hat
     Hat,
```

Comment:
What does the `Hat` operator mean? and why is there a separate `HatOp` operator?